test/helpers/smartcards.ts

```diff
@@ -45,9 +45,11 @@ export const pollWorkerCard: CardPresentAPI = {
 }
 
 export const createVoterCard = (
-  config?: Partial<VoterCardData>
+  config?: Partial<VoterCardData>,
+  longValueExists?: boolean
```

Comment:
I hear you, but this is used with an extra param in only one place, and changing this makes the rest of the code that doesn't use the second parameter a good bit uglier. I'd like to keep it as is for now.

Reply:
It could be added to `config`. Something like this:

```ts
type CreateVoterCardConfig = Partial<VoterCardData> & {
  longValueExists?: boolean
}

export const createVoterCard = (
  config: CreateVoterCardConfig = {}
): CardPresentAPI => {
  const { longValueExists, ...cardData } = config
  return {
    present: true,
    longValueExists,
    shortValue: JSON.stringify({
      t: 'voter',
      c: utcTimestamp(),
      pr: election.precincts[0].id,
      bs: election.ballotStyles[0].id,
      ...cardData,
    }),
  }
}
```